README.rst

```diff
@@ -602,6 +602,52 @@ in `/tmp/sops.sock` and not the local key service, you can run:
 
 `sops --enable-local-keyservice=false --keyservice unix:///tmp/sops.sock -d file.yaml`
 
+Auditing
+~~~~~~~~
+
+Sometimes, users want to be able to tell what files were accessed by whom in an
+environment they control. For this reason, SOPS can generate audit logs to
+record activity on encrypted files. When enabled, SOPS will write a log entry
+into a pre-configured PostgreSQL database when a file is decrypted. The log
+includes a timestamp, the username SOPS is running as, and the file that was
+decrypted.
+
+In order to enable auditing, you must first create the database and credentials
+using the schema found in :code:`audit/schema.sql`. This schema defines the
+tables that store the audit events and a role named :code:`sops` that only has
+permission to add entries to the audit event tables. The default password for
+the role :code:`sops` is :code:`sops`. You should change this password.
+
+Once you have created the database, you have to tell SOPS how to connect to it.
+Because we don't want users of SOPS to be able to control auditing, the audit
+configuration file location is not configurable, and must be at
+:code:`/etc/sops/audit.yaml`
```

Comment:
Add: `This file should have strict permissions such that only the root user can modify it.`